crates/bevy_pbr/src/lib.rs

```diff
@@ -136,11 +135,9 @@ impl Plugin for PbrPlugin {
             .register_type::<PointLightShadowMap>()
             .add_plugin(MeshRenderPlugin)
             .add_plugin(MaterialPlugin::<StandardMaterial>::default())
-            .init_resource::<AmbientLight>()
             .init_resource::<GlobalVisiblePointLights>()
             .init_resource::<DirectionalLightShadowMap>()
             .init_resource::<PointLightShadowMap>()
-            .add_plugin(ExtractResourcePlugin::<AmbientLight>::default())
```

Comment:
Still need to extract the `AmbientLight` component. Try this `.add_plugin(ExtractComponentPlugin::<AmbientLight>::default())`.

You'll probably need to write something similar to the following

```rust
impl ExtractComponent for Fxaa {
    type Query = &'static Self;
    type Filter = With<Camera>;
    type Out = Self;

    fn extract_component(item: QueryItem<Self::Query>) -> Option<Self> {
        Some(item.clone())
    }
}
```